db_benchmark_test.go

```diff
@@ -24,21 +24,94 @@ package main_test
 import (
 	"os"
 	"testing"
+	"time"
 
 	"database/sql"
 
+	"github.com/google/uuid"
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
 
 	main "github.com/RedHatInsights/ccx-notification-writer"
 )
 
+// Configuration-related constants
 const (
 	configFileEnvName = "CCX_NOTIFICATION_WRITER_CONFIG_FILE"
 	configFileName    = "tests/benchmark"
 )
 
-// initLogging function initializes logging
+// SQL statements
+//
+// Table reported V1 is reported table without event_type column and constraint for this column
+// Table reported V2 is reported table with event_type column and constraint for this column
+const (
+	dropTableReportedV1 = `DROP TABLE IF EXISTS reported_benchmark_1;`
+	dropTableReportedV2 = `DROP TABLE IF EXISTS reported_benchmark_2;`
+
+	createTableReportedV1 = `
+                CREATE TABLE IF NOT EXISTS reported_benchmark_1 (
+                    org_id            integer not null,
+                    account_number    integer not null,
+                    cluster           character(36) not null,
+                    notification_type integer not null,
+                    state             integer not null,
+                    report            varchar not null,
+                    updated_at        timestamp not null,
+                    notified_at       timestamp not null,
+		    error_log         varchar,
+                
+                    PRIMARY KEY (org_id, cluster, notified_at),
+                    CONSTRAINT fk_notification_type
+                        foreign key(notification_type)
+                        references notification_types(id),
```

Comment:
It is necessary to create these tables too:
- notification_types
- states
isn't it? I can't run the benchmarks locally.

Reply:
also, if you want to run this using compose, we have to change the admin password to `postgres` [here](https://github.com/RedHatInsights/ccx-notification-writer/blob/master/docker-compose.yml)

Reply:
it will run benchmarks, it's prepared to be run against migrated database

Reply:
OTOH yes, we might probably start the migration from benchmarks, if you really wish to

Reply:
hmm, if the migration is not included I'd add a brief explanation on what is needed for this benchmarks to run. Just a couple of lines in the code with the commands to run

Reply:
added